.github/workflows/run_tests.yml

```diff
@@ -50,6 +50,6 @@ jobs:
       - uses: "ramsey/composer-install@v2"
       - uses: paambaati/codeclimate-action@v5.0.0
         env:
-          CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
+          CC_TEST_REPORTER_ID: "${{ secrets.CC_TEST_REPORTER_ID }}"
```

Comment:
Why was this change needed?

Reply:
I dunno, it just stopped working for some reason without the quotes.